Fixes Quanton pin mapping when IN port is mapped as outputs

vladkvit opened the pull request:
When quanton IN pins are configured as outputs, the 9th and 10th outputs are now mapped onto IN1 and IN2 instead of IN2 and IN3.

Before, the code handled "PPM+Outputs" same as "Outputs" when configuring the IN port.

Commit:
When quanton IN pins are configured as outputs, the 9th and 10th outputs are mapped onto IN1 and IN2 instead of IN2 and IN3

mlyle commented:
jenkins, test this please

mlyle commented on flight/targets/quanton/board-info/board_hw_defs.c:
```diff
@@ -1646,7 +1646,284 @@ static const struct pios_tim_channel pios_tim_servoport_rcvrport_pins[] = {
+    16: TIM5_CH2  (PA1)		(IN8)
+ */
 
+static const struct pios_tim_channel pios_tim_servoport_rcvrport_pins[] = {
```
I've inspected the contents of this, and it looks good.

One comment though-- our standard is for C flight code to be indented per Linux kernel coding standards;  specifically, hte contents of this structure should be indented using tabs rather than spaces.

Once the structure is indented properly, I'll re-review and will be glad to merge.

vladkvit committed:
Spaces to tabs

NOTE: I only modified my additions. There are a few other spots that use spaces (e.g. line 132)

vladkvit committed:
Spaces to tabs

mlyle commented:
jenkins, test this please

vladkvit commented:
@mlyle Updated my code to tabs.

mlyle commented:
@vladkvit :) Thanks for the contribution (more outputs is nice, yay) and patience with coding style.   Running it through test now.

vladkvit commented:
@mlyle the patch doesn't add more outputs, at least not yet, it just makes the mapping make more sense. Full 16 outputs are coming up next :)

tracernz commented:
Yeah, ActuatorSettings etc. only supports 10 outputs.

mlyle commented:
Hmm.  Full outputs will be tricky, then, because of F1 memory situation.

mlyle commented:
stupid jenkins.

jenkins, test this please
